pkg/cli/apps.go

```diff
@@ -118,37 +118,29 @@ func Apps(rack sdk.Interface, c *stdcli.Context) error {
 func AppsCancel(rack sdk.Interface, c *stdcli.Context) error {
 	aname := coalesce(c.Arg(0), app(c))
 
-	c.Startf("Cancelling <app>%s</app>", aname)
+	c.Startf("Cancelling deployment of <app>%s</app>\n", aname)
 
-	if err := rack.AppCancel(aname); err != nil {
-		return err
-	}
+	rl, err := helpers.ReleaseLatest(rack, aname)
```

Comment:
`ReleaseLatest` is the proper method for retrieving all release information. `ReleaseList` doesn't populate the environment variables param.